From: Ben Hutchings <ben@decadent.org.uk>
Date: Mon, 07 Jul 2014 09:52:12 +0100
Subject: ptrace: Avoid ABI change in 3.14.10
Forwarded: not-needed

Hide a new #include from genksyms, which changed the symbol hashes for
most of networking.

--- a/include/linux/ptrace.h
+++ b/include/linux/ptrace.h
@@ -5,7 +5,9 @@
 #include <linux/sched.h>		/* For struct task_struct.  */
 #include <linux/err.h>			/* for IS_ERR_VALUE */
 #include <linux/bug.h>			/* For BUG_ON.  */
+#ifndef __GENKSYMS__
 #include <linux/pid_namespace.h>	/* For task_active_pid_ns.  */
+#endif
 #include <uapi/linux/ptrace.h>
 
 /*
